pkg/virt-controller/watch/snapshot/snapshot.go

```diff
@@ -279,6 +275,24 @@ func (ctrl *VMSnapshotController) updateVMSnapshotContent(content *snapshotv1.Vi
 
 	if ready && (contentCpy.Status.ReadyToUse == nil || !*contentCpy.Status.ReadyToUse) {
 		contentCpy.Status.CreationTime = currentTime()
+
+		source, err := ctrl.getSnapshotSource(vmSnapshot)
+		if err != nil {
+			return 0, err
+		}
+
+		frozen, err := source.Frozen()
+		if err != nil {
+			return 0, err
+		}
+
+		if frozen {
+			if err := source.Unfreeze(); err != nil {
+				return 0, err
+			}
+
+			ready = false
```

Comment:
why change ready to false?

Reply:
THe idea is to actually observe the VM being thawed before saying ready to use